README.md

```diff
@@ -59,11 +60,26 @@ Please consider to be **Open Source First**:
 
 ## 3) Team/Maintainer Must
 
-- Use and force two factor Authentication everywhere it is possible: github, npm, nuget, etc.
-- Tokens must be kept private
+Checklist for repository creation:
+
+- Use snake-case pattern to name your repository
```

Comment:
Repository name convention should more depends of the language.
SlimFaas is more powerful than slimfaas.

I'am not for a rule for that, you cannot manage fork Repository name.

Reply:
Good point, I removed the name convention rule.